pom.xml

```diff
@@ -100,6 +100,7 @@
         <module>inlong-sort-standalone</module>
         <module>inlong-dashboard</module>
         <module>inlong-distribution</module>
+        <module>manager-client-test</module>
```

Comment:
No need to add an inner module to the parent pom file.

Reply:
done